web/src/views/ProfileCreate.tsx

```diff
@@ -71,8 +72,10 @@ const ProfileCreate: React.FC = () => {
 
   useEffect(() => {
     async function wrap() {
-      const response = await api.getMinistry();
-      setMinistry(response.data);
+      const ministryResponse = await api.getMinistry();
+      const clusterResponse = await api.getCluster();
```

Comment:
This is returning ALL clusters (ARO for instance isn't being used any more, and it technically allows for someone to provision on Gold OR Gold DR which won't be support as I understand it.

Additionally, there is no role based control (API completes a findALL() via the end point) . As it stands, anyone can attempt to provision a project in KLAB for instance.

Reply:
ARO is there because it has been included as DB data reference. Let's confirm with the team and remove it if necessary.

The authorization changes can be found in https://github.com/bcgov/platform-services-registry/pull/372.

Reply:
> Let's confirm with the team and remove it if necessary.

https://chat.developer.gov.bc.ca/channel/devops-registry?msg=qnakiMHr2vYHXBgiE